native/src/modules/layout/containers/HeaderContainer.js

```diff
@@ -47,12 +49,8 @@ const mapStateToProps = (state: StateType, ownProps: OwnPropsType): StatePropsTy
   const routeName = ownProps.scene.route.name
   const simpleRouteShareUrl =
     typeof cityCode === 'string' && simpleRoutes.includes(routeName)
-      ? cityContentUrl({
-          cityCode,
-          languageCode: state.contentLanguage,
-          route: routeName,
-          path: null
-        })
+      // $FlowFixMe Flow does not understand life and just hates the whole world :/
+      ? urlFromRouteInformation({ cityCode, languageCode: state.contentLanguage, route: routeName })
```

Comment:
to get rid of the flow error you could write: 
```suggestion
      ? urlFromRouteInformation({ cityCode, languageCode: state.contentLanguage, route: (routeName: $Values<typeof simpleRoutes> })
```

Reply:
Nice, thanks :+1: